lib/libpathconv/abs2rel.3

```diff
@@ -93,6 +93,7 @@ The
 .Fa size
 argument is greater than zero but smaller than the length of the pathname plus 1.
 
```

Comment:
Removing this blank line would not change the ascii output, but it *does* change the html output (removing an empty `<p></p>` paragraph).

I therefore left that blank line alone; I might submit a "clean up useless whitespace" PR once all the syntax errors are fixed, but that's probably a month away.